mobile/src/main/java/ch/epfl/sdp/mobile/application/chess/parser/StringCombinators.kt

```diff
@@ -23,6 +25,36 @@ object StringCombinators {
   /** Parses the first digit of a [String], if it exists. */
   fun digit(): Parser<String, Int> = char().filter { it in '0'..'9' }.map { it - '0' }
 
+  /**
+   * Parses the first [Token] of a [String]
+   *
+   * @param delimiter the delimiter between each token
+   */
+  fun string(delimiter: Char = ' '): Parser<String, Token> = Parser {
```

Comment:
I think the name `token` would describes better what it does, instead of `string`.

Reply:
Fix in commit https://github.com/epfl-SDP/android/pull/280/commits/e5dd6733d5a1211cfed6e41538ea568c7e165cf0